swap/src/protocol/alice/event_loop.rs

```diff
@@ -118,6 +146,14 @@ impl EventLoop {
         Ok((driver, handle))
     }
 
+    pub fn clone_handle(&self) -> EventLoopHandle {
+        EventLoopHandle {
+            recv_encrypted_signature: self.recv_encrypted_signature.subscribe(),
+            send_transfer_proof: self.handle.send_transfer_proof.clone(),
+            recv_transfer_proof_ack: self.recv_transfer_proof_ack.subscribe(),
+        }
+    }
```

Comment:
Why not just implement `Clone`? You don't need to derive it, you can just implement it yourself.

Reply:
Hum, good point, the thing is that to call the `receiver` end of a broadcast channel you need the `sender` end.
I could keep both ends in an `EventLoopHandle` to allow the cloning.

But in terms of API, it does not really matter because it is used in the `EventLoop` to produce a new handle. So maybe I should change the name of the function instead to `new_handle`.

Reply:
Ah, sorry. I didn't realize that this function is on `EventLoop` and not `EventLoopHandle`. Makes total sense!